src/platform/profile/additional-info/gender.ts

```diff
@@ -0,0 +1,23 @@
+import { generateLazyValidator, JSONSchema, ValidateFunction } from '../../../validation'
+
+export enum Gender {
+  FEMALE = 'Female',
```

Comment:
This should be a free form string, not an enum

Reply:
Maybe all should be strings then? I followed the design
CC: @RominaMarchetti @olavra

Reply:
Data is discretized so it can be processed later with the different options instead of people misspelling or typing in different languages. For that we also have an "Other" option that allows the player to type whatever they one for almost all the additional fields

Reply:
@agusaldasoro please check that all the fields have a other value, and its free text for them

Reply:
I would also like to see the SDK being able to make use of this data: for examplo, in the pride month, the sceenes can render the appropiate flag of your gender or sexual orientation if the information is discretized. 
Other examples are scenes that could use nationalities and working status for professional connections.

Reply:
Thanks, done! We decided to have the list of the enum in the UI, but don't validate the values on the backend. As we have the enum we can parse them later.